### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by pbolduc**
This PR includes a Helm chart that deploys a stateful set. This chart was based off of [ZeroDownTime/kubezero clamav](https://git.zero-downtime.net/ZeroDownTime/kubezero/src/branch/master/charts/clamav) Helm chart. Changes were made to the chart to create a deployment that matches the deployment config template.

In addition, [clamdcheck.sh](https://github.com/Cisco-Talos/clamav-docker/blob/fdf806abc5b077a3bb639517d236a3790dfc70e8/clamav/1.0/alpine/scripts/clamdcheck.sh) script was added to be able to enable startupProbe and livenessProbe

### CODE REVIEW

**Review by KDNeufeld — APPROVED** [review]
Great work!

File: README.md [hunk 1]
```diff
@@ -17,6 +17,34 @@ The templates in the [openshift/templates](./openshift/templates) will build and
...
+```
+
+4. When the build is complete, new image stream tag should be avilable, `clamav` with a single image stream tag `clamav:latest`
+
+5. Deploy, if running locally be sure to set the clamav.freshclam.mirrors
+
+```
+cd charts
+helm install clamav clamav --set clamav.image=image-registry.openshift-image-registry.svc:5000/clamav-demo/clamav --set clamav.freshclam.mirrors=https://db.us.clamav.net
+```
```
Comment (marklise) [inline README.md:45]: This should not be the default as the entirety of openshift will get ratelimited (use something like https://clamav-mirror-6cdc9e-dev.apps.silver.devops.gov.bc.ca)  Reaching outside openshift to update clamav will ban us from hitting it too often.

**Review by marklise — CHANGES_REQUESTED** [review]
You'll want to update your mirror location - also, I noticed some of your commits are unverified.  We don't have branch protection on this repo, but it's always good to have your gpg setup on all machine you might commit from.

File: README.md (hunk 1 above)
Comment (pbolduc) [reply README.md:45]: This part of the readme says if you are running locally, ie OpenShift local.

As for the mirrors, they are defaulted to the bcgov mirror if the user does not override. In the configmap,

https://github.com/pbolduc/bcgov-clamav/blob/e542aefe26be203d025d69cb61c5f7cac89b4e7a/charts/clamav/templates/configmap.yaml#L691-L692

and in the default values file,

https://github.com/pbolduc/bcgov-clamav/blob/e542aefe26be203d025d69cb61c5f7cac89b4e7a/charts/clamav/values.yaml#L24-L26

Comment (pbolduc) [reply README.md]: The default mirror is `https://clamav-mirror.apps.silver.devops.gov.bc.ca`

**Comment by pbolduc** [comment]
I just pushed two changes. 

One removes the mirrors from the `freshclam.conf` file in the configs folder. The helm chart does not use this file.

The second one changes the mirrors configuration to be an array. Allows more flexibly if in the future, there is a silver and gold mirror. It defaults to a single mirror, `https://clamav-mirror.apps.silver.devops.gov.bc.ca`

**Review by marklise — APPROVED** [review]
Looks great! Thank you for the contribution! 🎉

**Pull request merged**